### DISCUSSION TITLE
Expand Subheadings

### DISCUSSION THREAD

**Discussion by W3stw0rldp4ndaWork** [retypeapp/retype#512] [General]
Hey everyone!

There is a content bar on the right side panel of your retype pages. These come pretty handy to quick access specific chapters.
I've noticed that they are limited to one subheading.

It works for 
1. HEADING
1.1 Heading
2. HEADING
2.1 Heading
3. HEADING

I'd like to expand it by 1-2 more subheadings, like this:
1. HEADING
1.1 Heading
1.1.1 Another Heading
1.1.1.1 More Subheading

Is this even possible?
Thanks forward!

**Comment by DannyBen** [comment]
Discussed here: https://github.com/retypeapp/retype/discussions/355#discussioncomment-3087813

(not supported at the moment).

**Comment by geoffreymcgill** [comment]
As noted by @DannyBen, Retype will only generate a table-of-contents for the H2 and H3 heading levels, although the sample you provided only has two levels of headings, so it could be reproduced using the following Markdown:

```md
## 1. HEADING

The 1. Heading paragraph.

### 1.1 Heading

The 1.1 Heading paragraph.

### 1.1.1 Another Heading

The 1.1.1 Another Heading paragraph.

### 1.1.1.1 More Subheading

The 1.1.1.1 More Subheading paragraph.
```
